### PR TITLE
composition over context by Prateek Gogia

### PR DESCRIPTION
**Pull request by reeversedev**
This PR aims to add link for talk by @reeversedev on topic: "Compositon over context in React"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ We try to keep the list up-to-date.
 Please open a Pull Request to add a new public contribution.
 
 # 2022
+* March 12, 2022 [Composition over Context - Prateek Gogia](https://youtu.be/IJZFtwZgXd0)
```
Comment (kajolnigam) [inline README.md:9]:
```
[The Numbers Behind High Performing Engineering Teams with Varun Achar](https://www.treebotechtalks.com/posts/numbers-behind-high-performance-engineering-teams) - [Varun Achar](https://github.com/varunachar)
```

use this format

**Review by kajolnigam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (reeversedev) [reply README.md:9]: I don't have link for actual code. Just the YouTube video link.

Comment (kajolnigam) [reply README.md:9]: I meant, move your name and add github link at the end of entry

Comment (reeversedev) [reply README.md]: added the changes

**Review by kajolnigam — APPROVED** [review]

**Pull request merged**